mender-deb-package

```diff
@@ -37,7 +37,7 @@ verify_script_arguments() {
 checkout_repo_clean_local_path() {
   local -r repo_url="$1"
   local repo_path="${repo_url#https://}"
-  if [[ -n "${MENDER_PRIVATE_REPO_ACCESS_USER}" ]]; then
+  if [[ "$repo_url" == *"${MENDER_PRIVATE_REPO_ACCESS_USER:-none}"* ]]; then
```

Comment:
Question:
If the `MENDER_PRIVATE_REPO_ACCESS_USER` is defined for all calls, then why the default `:-none`?

Reply:
Good question. It is defined for all calls when running in master, not when running on a PR branch nor locally. (I didn't had the `none` default in my first attempt and then ironically the build fail on the PR build).